### ISSUE TITLE
Timezone question

### ISSUE THREAD

**Issue by dahall** [dahall/TaskScheduler#238]
Hi,

I'm creating a task that runs at a specific day of the week, at a specific time of the day.

When creating this task on computer "A" (a server), from TaskScheduler code running on computer "B" (my laptop), if the two computers are in different time zones, can anyone tell me whether the task will be scheduled using Computer "A"'s local date/time settings, or Computer "B"'s local date/time settings?

Basically, if my pc is in the Eastern time zone, and I schedule a task from my pc to run at 6 am on a computer in Central Standard time, will the job be scheduled at 6AM EST, or 6AM CST?

Thanks for your help!

Originally posted: 2009-08-14T08:46:30
